src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -2069,6 +2069,42 @@ inline RppStatus custom_convolve_image_host(T* srcPtr, RppiSize srcSize, U* dstP
 
 // Compute Functions for RPP Tensor API
 
+inline RppStatus compute_contrast_48_host(__m256 *p, __m256 *pContrastParams)
```

Comment:
do we really need to return RppStatus in these inline functions? Compiler will just replace functions with the code. Please make those void

Reply:
To be changed in a separate PR as discussed - following the merge of #104 #105 and #106